bigtable/admin/column_family.h

```diff
@@ -44,24 +44,27 @@ class GcRule {
    * Return a garbage collection rule that deletes cells in a column older than
    * the given duration.
    *
-   * The function accepts any instantiation of std::chrono::duration<> for the
+   * The function accepts any instantiation of `std::chrono::duration<>` for the
    * @p duration parameter.  For example:
    *
    * @code
    * auto rule1 = bigtable::GcRule::MaxAge(std::chrono::hours(48));
    * auto rule2 = bigtable::GcRule::MaxAge(std::chrono::seconds(48 * 3600));
-   * @encode
+   * @endcode
```

Comment:
I think this typo may be causing a build failure for the docs build; can you please move this (and similar change below) to the other PR to ensure it fixes it, and keep this change just for adding no-testing-needed code formatting?

Reply:
Done in #113.  Please merge that one first, I will rebase and resubmit.